gcexport.py

```diff
@@ -652,28 +719,28 @@ def export_data_file(activity_id, activity_details, args, file_time, append_desc
         makedirs(directory)
 
     # timestamp as prefix for filename
-    if args.fileprefix > 0:
-        prefix = "{}-".format(start_time_locale.replace("-", "").replace(":", b"").replace(" ", "-"))
+    if args.fileprefix and args.fileprefix > 0:
```

Comment:
This looks buggy, or buggy in the future. What was a purpose to change this line from the origin? I guess there is a case when the old one provides wrong results, could you please show the case?

Reply:
This change was a consequence from running `python2 -3` before the first tries with python3, I didn't really try if it's needed with python3

Reply:
Are you sure? Adding a logic with `and` has nothing to do with a compatibility. What I'm pointing here is the `args.fileprefix and`.